src/execution/riscv64/simulator-riscv64.cc

```diff
@@ -83,7 +83,7 @@ uint32_t get_fcsr_condition_bit(uint32_t cc) {
 // field of a subsequent LUI instruction; otherwise returns -1
 static inline int32_t get_ebreak_code(Instruction* instr) {
   DCHECK(instr->InstructionBits() == kBreakInstr);
-  Instruction* next_instr = instr + 1;
+  Instruction* next_instr = instr + kInstrSize;
```

Comment:
`instr + kInstrSize` would be to add `sizeof(Instruction)*kInstrSize` to pointer `instr`. I think the code should add `kInstrSize` to `sim_->pc` which is of byte type. Logically, it would be something like this, 
```
byte* curr_pc = ...;
Instruction* next_instr = reinterpret_cast<Instruction*>(curr_pc + kInstrSize);
```
but since this is a static method, you probably need to pass the next instruction to `get_ebreak_code()` as well, or make `get_ebreak_code` a method of `Simulator` so that it can access the PC.

Reply:
Thank you for your review.

I got it. I would remind myself about the result of `sizeof` in future.